content/soarian/dstu2/medications/immunization.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Immunization | Soarian DSTU 2 API
+---
+ 
+# Immunization
+
+* TOC
+{:toc}
+
+## Overview
+
+For the Immunization resource query, the API returns a consolidated list of immunizations charted as complete (administered, not administered or historical) for the patient.  The list may include immunizations administered internally and/or those externally reported as administered, including records reconciled from external sources such as the patient, state registries and/or consolidated clinical document architecture (C-CDA) documents.  The API does not return immunizations that may be ordered or scheduled but not yet administered.
+
+The information available for immunization records may vary depending on their source; for example, immunizations administered locally or as reconciled from a registry record may include ingredient details, codes, precise administration dates, etc. while an immunization reported by a patient or as received in a C-CDA may not.  The API will return the level of detail charted.  
+
+The dates associated with immunizations may reflect both charted  dates and administration dates which are most likely the same for internally administered records but will vary for externally sourced records.  Further, administration dates may vary in precision and accuracy depending on the source; specifically patient expressed dates may be fuzzy dates.  To avoid ambiguity of intended date context, the API will always return all configured and charted records; no date/date range parameters are supported.
+
+## Terminology Bindings
+
+<%= terminology_table(:soarian_immunization, :dstu2) %>
+
+## Custom Extensions
+
+The [bundle extension] can be returned, see possible codes in the [errors section] below.
+
+## Search
+
+	GET /Immunization?:parameters
+
+_Implementation Notes_
+
+* The API returns the most current immunization list.
```

Comment:
It needs to be added to implementation notes, I didn't catch that this specific code is the only returned, but either way, that's important enough that it should be specifically called out in the limitations/implementation notes